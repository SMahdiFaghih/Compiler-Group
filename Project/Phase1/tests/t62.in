"hi" + "int a =     blahblahblah ; ____X0x.2"