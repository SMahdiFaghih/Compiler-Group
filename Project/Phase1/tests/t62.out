T_STRINGLITERAL "hi"
+
T_STRINGLITERAL "int a =     blahblahblah ; ____X0x.2"
